tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -138,7 +134,7 @@ spec:
               fi
             fi
             # Push the associated source container using the common tag
-            if [[ "$(params.pushSourceContainer)" == "true" ]] ; then # Default to false
+            if [[ $(jq -r ".images.pushSourceContainer" "${DATA_FILE}") != "true" ]] ; then # Default to false
```

Comment:
I think you want != false here... otherwise this is executing when it is false or not provided, which seems opposite of what we want

Reply:
as we have above `if [[ $(jq -r ".images.addTimestampTag" "${DATA_FILE}") == "true" ]] ; then # Default to false`